File: backend/src/main/java/br/com/restaurante/gestao_restaurante/dto/pedido/PedidoResponseDTO.java
package br.com.restaurante.gestao_restaurante.dto.pedido;

import br.com.restaurante.gestao_restaurante.dto.item.ItemResponseDTO;
import lombok.Getter;
import lombok.Setter;

@Getter
@Setter
public class PedidoResponseDTO {
    private Long id;
    private Long comandaId;
    private Long garcomId;
    private Long itemId;
    private Long cozinheiroId;
    private Long barmanId;
    private Integer quantidade;
    private String obs;
    private String status;
    private Integer mesaNumero;
    @com.fasterxml.jackson.annotation.JsonFormat(pattern = "yyyy-MM-dd'T'HH:mm:ss")
    private java.time.LocalDateTime dataHora;
    private ItemResponseDTO item;
}

File: frontend/src/dto/pedido/PedidoResponseDTO.ts
import { ItemResponseDTO } from '../item/ItemResponseDTO';

export interface PedidoResponseDTO {
    id: number;
    comandaId: number;
    garcomId: number;
    itemId: number;
    cozinheiroId?: number;
    barmanId?: number;
    quantidade: number;
    obs?: string;
    status: string;
    mesaNumero?: number;
    dataHora: string;
    item: ItemResponseDTO;
}
